WordPress/Classes/Services/SiteManagementService.swift

```diff
@@ -31,6 +31,9 @@ open class SiteManagementService: LocalCoreDataService {
                     let blogService = BlogService(managedObjectContext: self.managedObjectContext)
                     blogService.remove(blog)
 
+                    let scheduler = try? BloggingRemindersScheduler()
```

Comment:
Since the call above to `blogService.remove(blog)` is already taking care of the unscheduling, should we remove this call?